app/src/main/java/org/mozilla/fenix/crashes/CrashContentInteractor.kt

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.crashes
+
+import android.util.Log
+import androidx.annotation.VisibleForTesting
+import kotlinx.coroutines.DelicateCoroutinesApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import org.mozilla.fenix.components.AppStore
+import org.mozilla.fenix.components.Components
+import org.mozilla.fenix.components.appstate.AppAction
+import org.mozilla.fenix.utils.Settings
+
+/**
+ * Handles all user interactions with a [CrashContentView].
+ */
+class CrashContentInteractor(
```

Comment:
Went with just an Interactor for composables and not delegating to a Controller following the model from https://github.com/mozilla-mobile/focus-android/pull/5985#issuecomment-986848567

Reply:
NA if using XML.